### PR TITLE
Optional image partition resize

### PR DESCRIPTION
**Pull request by tpatzig**
- custom cloud iso path and name
- custom image folder
- allow clouddata fqdn
- allow to skip image partition resize

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -250,6 +250,10 @@ function sshrun
         export scenario=$scenario ;
         export host_mtu=$host_mtu ;
         export architectures='$architectures';
+        export cloud6_iso_path=$cloud6_iso_path ;
```
Comment (jdsn) [inline scripts/mkcloud]: these are candidates for the `want_` variables
just name them `want_iso_xxx` and they will be passed to the admin node via magic

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1209,16 +1209,16 @@ function onadmin_set_source_variables
             CLOUDLOCALREPOS="SUSE-Cloud-5-devel"
         ;;
         develcloud6)
-            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6/images/iso
+            CLOUDSLE12DISTPATH=${cloud6_iso_path:='/ibs/Devel:/Cloud:/6/images/iso'}
             [ -n "$TESTHEAD" ] && CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6:/Staging/images/iso
-            CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-6-$arch*Media1.iso"
+            CLOUDSLE12DISTISO=${cloud6_iso:="SUSE-OPENSTACK-CLOUD-6-$arch*Media1.iso"}
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: The default value is duplicated here. IMHO it is enough to have it in one place.

File: scripts/mkcloud (hunk 1 above)
Comment (bmwiedemann) [inline scripts/mkcloud]: grep=magic - I always knew it ;-)

**Comment by jdsn** [comment]
Btw. these 4 commits are not related, so its rather 4 individual pull requests.
This helps to discuss and merge them separately (because one issue could block all commits from being merged fast).

We cherry-picked commit 2 and 4 - so please rebase this PR - and maybe split it into two.

File: scripts/qa_crowbarsetup.sh (hunk 2 above)
Comment (tpatzig) [inline scripts/qa_crowbarsetup.sh]: right. i will remove, that default value.

**Comment by tpatzig** [comment]
thanks for cherry-picking. i will create 2 new pr, and close that then.

**Pull request closed without merging**